week-2/E-conditionals/exercise.js

```diff
@@ -11,10 +11,11 @@ var role = "mentor";
 var name2 = "Harun";
 var role2 = "student";
 
-
+if (role=="student"){
 console.log("Hi, My name is " + name);
+} else {  
 console.log("Hi, My name is " + name2);
-
+}
```

Comment:
This matches the expected result, but it wouldn't if I switched lines 15 and 17. We're looking for you to check the role of each person like this:

```
if (role === "student") {
  console.log("Hi, My name is " + name);
}
if (role2 === "student") {
  console.log("Hi, My name is " + name2);
}
```